### DISCUSSION TITLE
Can we indicate a user is from "UGent Memorialis" and "former UGent"?

### DISCUSSION THREAD

**Discussion by mietcls** [ugent-library/biblio-backend#1186] [General]
# Enhancement

1. Can we indicate a user is from "UGent Memorialis"?

For instance, if you look up http://www.ugentmemorialis.be/catalog/000008316 in Biblio, we could indicate that these people are part of the memorials. This would help people to figure out quicker who belongs to what research.

Right now, it looks like this:
![Screenshot 2023-08-21 at 11 55 19](https://github.com/ugent-library/biblio-backoffice/assets/1572885/781d4726-5346-4f9e-bd10-e2aa942492c8)

We could do something like this:
![Screenshot 2023-08-21 at 11 57 18](https://github.com/ugent-library/biblio-backoffice/assets/1572885/d28b4092-1819-4374-86f3-7f7313884eb1)

2. If a researcher *was* at UGent but their career came to an end, can we also indicate this?

3. What information do we retain for former UGent employees?

We would need to keep department around as well.

**Comment by mietcls** [comment]
I think this is one for @nicolasfranck in light of the person service.

**Reply by nicolasfranck** [reply]
* the search index as filled by the biblio frontend does not add ugent memorialis id right now. Could be added
* the authority database in the old authority database does not store previous affiliations, only the latest update it got. By the way, we also do not store the affiliation of the author in relation to the publication (could be different from his/her current affiliation). That remains the same for the people service in development. Especially is loading of "live person information" is required, that would conflict.
